readme.md

```diff
@@ -142,6 +142,7 @@
 - [ESP](https://github.com/agucova/awesome-esp#readme) - Low-cost microcontrollers with WiFi and broad IoT applications.
 - [Deno](https://github.com/denolib/awesome-deno#readme) - A secure runtime for JavaScript and TypeScript that uses V8 and is built in Rust.
 - [DOS](https://github.com/balintkissdev/awesome-dos#readme) - Operating system for x86-based personal computers that was popular during the 1980s and early 1990s.
+- [Nix](https://github.com/nix-community/awesome-nix#readme) - The reliable and reproduceable package manager for Linux and other Unix Systems.
```

Comment:
I think the description at the top of the awesome-nix readme is better.

Reply:
Just thought it might be a little long. I can change it pretty quick though.

Reply:
Changed in c5813c3